worker/introspection/script.go

```diff
@@ -131,6 +117,29 @@ juju_machine_lock () {
   done
 }
 
+juju_unit_status () {
+  juju_agent units?action=status
+}
+
+juju_stop_unit () {
+  arr=("$@")
+  local -a args
+  for i in "${arr[@]}"; do
+    args+=("unit=$i")
+  done
+  juju_agent --post units action=stop "${args[@]}"
```

Comment:
should check that the unit to stop is actually passed as an arg to the bash function
same with start
the handler will error with "missing unit" but nice/better to do it here also to avoid the unnecessary round trip

Reply:
Done.